lambdaclass/champions_of_mirra
# Connect AFK Rewards

Closes #337 

⚠️ Merge together with this PR on the back-end: https://github.com/lambdaclass/mirra_backend/pull/474

## Summary of changes

- Get the AFK Rewards and AFK Rewards Rate from the backend. 
- Claim AFK Rewards and update the user accordingly.
- Start using the Experience as a currency, to be more consistent with what was already implemented (this has no visible effects, just check the code and make sure that the battle rewards are still working as intended).

## How has this been tested?

1. Start the game with this branch on the back-end: https://github.com/lambdaclass/mirra_backend/pull/474
2. Open the AFK Rewards pop up and check that everything is 0.
3. Fight and win a battle, so you start getting AFK Rewards.
4. Go back to the AFK Rewards pop-up and see how you are now getting rewards. 
5. Claim the rewards, and verify that the user receives the awarded currencies (you can see how the gold increases in the header).

## Checklist
